ui/src/app/applications/components/application-create-panel/application-create-panel.scss

```diff
@@ -12,3 +12,9 @@
         padding-top: 5px;
     }
 }
+
+.pre, pre {
```

Comment:
get feedback from Alex M

Reply:
please move it into .application-create-panel:

```
.application-create-panel {
   .pre, pre {
      font-family: monospace;
      line-height: normal;
      white-space: pre;
  } 
}
```

unfortunatelly all styles are loaded into global scope. Same in `application-details.scss`